app/routes/locations.py

```diff
@@ -0,0 +1,81 @@
+from flask import Blueprint, url_for, redirect, request, jsonify
+from app.models import db, Location, Amenities, Necessities
+
+bp = Blueprint("location", __name__, url_prefix="/location")
+
+@bp.route("/", methods=["GET"])
+def get_all_locations():
+    locations = Location.query.all()
+    return jsonify(locations)
+
+@bp.route("/", methods=["GET"])
+def get_location(location_id):
+    # get location by location_id
+    location = Location.query.filter_by(location_id).one()
```

Comment:
out of curiosity, why not using get method instead of filter_by?
example: Location.query.get(location_id)

Reply:
I am still familiarizing myself with the query methods in python. I used filter_by in a tutorial. I have no strong preference. We should use the get method if the syntax on line 14 does not work.